src/Context/HelperContext.php

```diff
@@ -48,6 +48,10 @@ class HelperContext extends RawDrupalContext implements SnippetAcceptingContext
   // Date format.
   const DATE_FORMAT_CONCISE = "dmY-His";
 
+  //  File extensions.
+  const EXTENSION_HTML = '.html';
```

Comment:
Nicer if you keep the dot separate, so there is a change this can be used for more.